db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/add_provider.rb

```diff
@@ -1,24 +1,12 @@
-require 'net/ssh'
-
 DEPLOYMENT_TYPES = {
-  :origin            => "ManageIQ::Providers::Openshift::ContainerManager",
-  :enterprise        => "ManageIQ::Providers::OpenshiftEnterprise::ContainerManager",
-  :atomic            => "ManageIQ::Providers::Atomic::ContainerManager",
-  :atomic_enterprise => "ManageIQ::Providers::AtomicEnterprise::ContainerManager"
+  :origin               => "ManageIQ::Providers::Openshift::ContainerManager",
+  :openshift_enterprise => "ManageIQ::Providers::OpenshiftEnterprise::ContainerManager",
+  :atomic               => "ManageIQ::Providers::Atomic::ContainerManager",
+  :atomic_enterprise    => "ManageIQ::Providers::AtomicEnterprise::ContainerManager"
 }. freeze
 
 def provider_token
-  token = ""
-  Net::SSH.start($evm.root['deployment_master'], $evm.root['user'], :paranoid => false, :forward_agent => true,
-                 :key_data => $evm.root['private_key']) do |ssh|
-    cmd = "oc get -n management-infra sa/management-admin --template='{{range .secrets}}{{printf " + '"%s\n"' \
-          " .name}}{{end}}' | grep token"
-    token_key = ssh.exec!(cmd)
-    token_key.delete!("\n")
-    cmd = "oc get -n management-infra secrets #{token_key} --template='{{.data.token}}' | base64 -d"
-    token = ssh.exec!(cmd)
-  end
-  token
+  deployment.perform_agent_commands($evm.root['deployment_master'], $evm.root['ssh_username'], commands)[:stdout]
```

Comment:
https://github.com/ManageIQ/manageiq/pull/9230/files#diff-6e045b8ce93dfa6ccec7115f7cd1cba0R49
Its done in this pr :)